src/bzflag/defaultBZDB.cxx

```diff
@@ -34,15 +34,15 @@ DefaultDBItem defaultDBItems[] =
     { "volume",           "10",           true,   StateDatabase::ReadWrite,   NULL },
     { "latitude",         "37.5",         true,   StateDatabase::ReadWrite,   NULL },
     { "longitude",        "122",          true,   StateDatabase::ReadWrite,   NULL },
-    { "radarStyle",       "3",            true,   StateDatabase::ReadWrite,   NULL },
+    { "radarStyle",       "2",            true,   StateDatabase::ReadWrite,   NULL },
```

Comment:
Are the colors used in our "fast sorted" radar style standardized amongst other games? Otherwise, I'd find it confusing for new players to understand what all of the colors mean.

Reply:
From my experience, most games do not having a concept of depth in their mini-maps, short of maybe an indication that another entity is above/below the player, or maybe some sort of 3D-ish radar in a space battle game.  Most games have some static top-view representation of the world and overlay icons on top of that to indicate player/object/waypoint locations.

Reply:
Hmm then my personal preference for a default radar style would "Enhanced" since it shows depth just with different alpha values instead of introducing greens, reds, purples, etc.

Reply:
I agree.  I also just checked and death physics are only indicated on Enhanced radar.  I'll make the change shortly.